checklist_plus/text_generation/llm.py

```
@@ -0,0 +1,688 @@
+
+                # Ensure we have the requested number of paraphrases
+                if len(filtered_paraphrases) < n_paraphrases:
+                    # If we don't have enough unique paraphrases, pad with what we have
+                    while len(filtered_paraphrases) < n_paraphrases and filtered_paraphrases:
+                        filtered_paraphrases.append(filtered_paraphrases[0])
+
+                all_paraphrases.extend(filtered_paraphrases[:n_paraphrases])
+
+        except Exception as e:
+            print(f"LLM batch paraphrase failed: {e}")
+            raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)